test/verify/check-storage-btrfs

```diff
@@ -0,0 +1,560 @@
+
+        # test link to volume
+        b.click(self.card_button("btrfs device", label))
+        b.wait_text(self.card_desc("btrfs volume", "Label"), label)
+
+        # Format the btrfs device
+        b.go("#/")
+        b.wait_visible(self.card("Storage"))
+        self.click_dropdown(self.card_row("Storage", name="sda"), "Format")
+        self.dialog_wait_open()
+        self.dialog_wait_title(f"{dev_1} is in use")
+        self.dialog_cancel()
+        self.dialog_wait_close()
+
+        self.click_dropdown(self.card_row("Storage", location=subvol_mount_point), "Unmount")
+        self.confirm()
+
+        self.click_dropdown(self.card_row("Storage", location=mount_point), "Unmount")
+        self.confirm()
+        b.wait_visible(self.card_row("Storage", location=f"{mount_point} (not mounted)"))
+
+        # Also when not mounted
+        self.click_dropdown(self.card_row("Storage", name="sda"), "Format")
+        self.dialog_wait_open()
+        self.dialog_wait_title(f"{dev_1} is in use")
+        self.dialog_cancel()
+        self.dialog_wait_close()
+
+    def testMultiDevice(self):
+        m = self.machine
+        b = self.browser
+
+        self.login_and_go("/storage")
+
+        disk1 = self.add_ram_disk(size=140)
+        disk2 = self.add_loopback_disk(size=140)
+        label = "raid1"
+        mount_point = "/run/butter"
+        subvol_mount_point = "/run/cake"
+        subvol = "/run/butter/cake"
+        subvol2 = "/run/butter/bread"
+        subvol_name = os.path.basename(subvol)
+
+        m.execute(f"mkfs.btrfs -L {label} -d raid1 {disk1} {disk2}")
+        self.login_and_go("/storage")
+
+        # creation of btrfs partition can take a while on TF.
+        with b.wait_timeout(30):
+            b.wait_visible(self.card_row("Storage", name=label))
+
+        b.wait_in_text(self.card_row("Storage", name=os.path.basename(disk1)), "btrfs device")
+        b.wait_in_text(self.card_row("Storage", name=os.path.basename(disk2)), "btrfs device")
+
+        # mount /
+        self.click_dropdown(self.card_row("Storage", name=label) + " + tr", "Mount")
+        self.dialog({"mount_point": mount_point})
+        b.wait_visible(self.card_row("Storage", location=mount_point))
+
+        # create subvolume
+        m.execute(f"""
+            btrfs subvolume create {subvol}
+            btrfs subvolume create {subvol2}
+        """)
+        b.wait_visible(self.card_row("Storage", name=os.path.basename(subvol)))
+
+        self.click_dropdown(self.card_row("Storage", name=os.path.basename(subvol)), "Mount")
+        self.dialog({"mount_point": subvol_mount_point})
+        b.wait_visible(self.card_row("Storage", location=subvol_mount_point))
+
+        # devices overview
+        self.click_card_row("Storage", name=label)
+        b.wait_visible(self.card_row("btrfs volume", name=disk1))
+        b.wait_visible(self.card_row("btrfs volume", name=disk2))
+
+        # Allocate some space to see if the space is spread evenly
```

Comment:
I consider skipping this one as it would require btrfs balancing. In essence we show the per device usage of a btrfs device

```
[root@fedora-39-127-0-0-2-2201 ~]# btrfs filesystem show
Label: 'fedora'  uuid: cece4dd8-6168-4c88-a4a8-f7c51ed4f82b
        Total devices 1 FS bytes used 1.89GiB
        devid    1 size 11.92GiB used 3.51GiB path /dev/vda5

Label: 'raid1'  uuid: ba734eaa-a306-4015-a43b-aa30db10d098
        Total devices 2 FS bytes used 176.00KiB
        devid    1 size 140.00MiB used 104.00MiB path /dev/sda
        devid    2 size 133.51MiB used 104.00MiB path /dev/loop0

[root@fedora-39-127-0-0-2-2201 ~]# btrfs filesystem df /run/butter
Data, RAID1: total=64.00MiB, used=0.00B
System, RAID1: total=8.00MiB, used=16.00KiB
Metadata, RAID1: total=32.00MiB, used=160.00KiB
GlobalReserve, single: total=5.50MiB, used=0.00B
[root@fedora-39-127-0-0-2-2201 ~]#
```

We show 104 MiB used which corresponds to what btrfs gives us. Basically these disk sizes are too small to create any kind of meaningful test. I'd like to do that in a follow up. I am not sure how big we want to grow these ram disks after all. I'll handle this in a follow up which focuses more on multi disk devices.